src/printer.js

```diff
@@ -65,12 +64,14 @@ Printer.prototype.print = function(callback) {
 	async.eachSeries(
 		_self.commandQueue,
 		function(command, callback) {
-			if (_self.commandDelay !== 0) {
-				sleep.usleep(_self.commandDelay);
+			function write() {
+				_self.serialPort.write(command, function() {
+					_self.serialPort.drain(callback);
+				});
 			}
-			_self.serialPort.write(command, function() {
-				_self.serialPort.drain(callback);
-			});
+
+			if (_self.commandDelay === 0) return write()
```

Comment:
I'll updated to that then.

Have you considered following the [JavaScript Standard Style](https://github.com/feross/standard)?